lib/very_good_infinite_list.dart

```diff
@@ -328,7 +326,7 @@ extension on _ListStatus {
 
 class _ListException implements Exception {
   const _ListException(this.value);
-  final Object value;
+  final Object? value;
```

Comment:
can we make this non-nullable?

Reply:
If we want to do this which we should then we have to handle this in some other way.

<img width="416" alt="image" src="https://user-images.githubusercontent.com/56479665/107902898-c27ca880-6f6d-11eb-858f-fd672ab3dbb4.png">

Reply:
Might be something like this 
<img width="380" alt="image" src="https://user-images.githubusercontent.com/56479665/107902955-ef30c000-6f6d-11eb-96df-03d7e8cbb36e.png">

Reply:
we can do `static const none = Object();`

Reply:
Then we are getting this error
<img width="854" alt="image" src="https://user-images.githubusercontent.com/56479665/107904189-0f15b300-6f71-11eb-991c-183f50ff597c.png">

Reply:
```static const none = _ListException(Object());```
should we do something like this ?

Reply:
yeah sorry that's what I meant